lib/filesystem/CFilesystemLoader.cpp

```diff
@@ -88,7 +88,7 @@ bool CFilesystemLoader::createResource(std::string filename, bool update)
 
 	if (!update)
 	{
-		if (!FileStream::CreateFile(baseDirectory / filename))
+		if (!FileStream::FileCreate(baseDirectory / filename))
```

Comment:
Ah, hello, yet another macro from Windows.h breaking our builds.
I would rather rename it to createFile, since we use camelCase for methods, but no big deal

Reply:
Fixed.